mt-kahypar/datastructures/static_hypergraph.cpp

```diff
@@ -414,11 +417,11 @@ namespace mt_kahypar::ds {
       // Write hyperedges from temporary buffers to incidence array
       tbb::enumerable_thread_specific<size_t> local_max_edge_size(UL(0));
       tbb::parallel_for(ID(0), _num_hyperedges, [&](const HyperedgeID& id) {
-        if ( he_mapping.value(id) /* hyperedge is valid */ ) {
+        if ( he_mapping.value(id) > 0 /* hyperedge is valid */ ) {
           const size_t he_pos = he_mapping[id];
           const size_t incidence_array_start = num_pins_prefix_sum[id];
           Hyperedge& he = hypergraph._hyperedges[he_pos];
-          he = std::move(tmp_hyperedges[id]);
+          he = tmp_hyperedges[id];
```

Comment:
Why removing std::move(...) here?

Reply:
Because it's slightly misleading. The type Hyperedge consists only of POD types so it's just a copy.